htdocs/core/lib/date.lib.php

```diff
@@ -286,7 +286,7 @@ function convertSecondToTime($iSecond, $format = 'all', $lengthOfDay = 86400, $l
 			if ($sDay > 1) {
 				$dayTranslate = $langs->trans("Days");
 			}
-			$sTime .= $sDay.' '.strtolower(dol_substr($dayTranslate, 0, 1)).'. ';
+			$sTime .= $sDay.' '.$dayTranslate.' ';
```

Comment:
Hi we already have the translation for first letter of day into language file, into product.lang we have for english
d=d
for french
d=j

I suppose you need d=T for german ?  So i suggest to fix with 

$langs->load("products");
$sTime .= $sDay.' '.$langs->trans("d").' ';

Then to fix the german translation for d into transifex...

Reply:
Hi,
that's a perfect solution. Actually, the one letter abbreviation for Day in German is d=d (coming from dies in Latin for whatever reason...). I just checked in Transifex and the translation was already ok.